staff/pere-hernandez/git/README.md

```diff
@@ -0,0 +1,70 @@
+# GIT
+
+Command-line interface comands typical in GIT
+
+## git log --- Shows a list of commits ---
+
+```sh
+$ git log
+```
+
+## git log --graph --- Shows a list of branches in a line ---
+
+```sh
+$ git log --graph
+```
+
+## git branch --- Shows all the current branches ---
+
+```sh
+$ git branch
+```
+
+## git branch -D feature/git --- Deletes the branch 'feature/git' ---
+
+```sh
+$ git branch -D feature/git
+```
+
+## git branch develop --- Creates a new branch called 'develop' ---
+
+```sh
+$ git branch develop
+```
+
+## git checkout develop --- Moves to a branch named 'develop' ---
+
+```sh
+$ git checkout develop
+```
+
+## git add develop --- Updates the info in the branch named 'develop' to prepare it for commit ---
```

Comment:
The git add command doesn't apply to branches directly, but to directories or files. Correct the definition in order to be more clear 🙀

Reply:
👻🚀🚀🚀